core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -126,4 +126,8 @@ private TableProperties() {
 
   public static final String SNAPSHOT_ID_INHERITANCE_ENABLED = "compatibility.snapshot-id-inheritance.enabled";
   public static final boolean SNAPSHOT_ID_INHERITANCE_ENABLED_DEFAULT = false;
+
+  public static final String ENGINE_HIVE_ENABLED = "engine.hive.enabled";
+  public static final String ENGINE_HIVE_ENABLED_HIVE_CONF = "iceberg.engine.hive.enabled";
```

Comment:
Could we move this to a class for Hadoop configuration properties, like `org.apache.iceberg.hadoop.ConfigProperties`?

Reply:
Done